duke-core/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.classpath
+/.settings/
```

Comment:
I think we should update root .gitignore file with correct patterns and avoid maintaining .gitgnore for each submodule